integration-libs/cdc/root/consent-management/services/cdc-consent-management.service.ts

```diff
@@ -0,0 +1,49 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable } from '@angular/core';
+import { ConsentTemplate } from '@spartacus/core';
+import { ConsentManagementService } from '@spartacus/storefront';
+import { CdcLocalStorageTemplate } from '../model/index';
+import { CdcConsentsLocalStorageService } from './cdc-consents-local-storage.service';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class CdcConsentManagementService extends ConsentManagementService {
+  constructor(protected store: CdcConsentsLocalStorageService) {
+    super();
+  }
+  getRequiredConsents(templateList: ConsentTemplate[]): string[] {
+    let requiredConsents: string[] = [];
+    const cdcConsents = this.getCdcConsentIDs(true);
+    requiredConsents = super.getRequiredConsents(templateList);
+    requiredConsents.push(...cdcConsents);
```

Comment:
Let's not mutate the original array coming from the service (which I understand is the array taken from the static anonymousConsentsConfig -> in other words, let's not mutate accidentaly the static config).
```suggestion
    requiredConsents.push(...super.getRequiredConsents(templateList));
    requiredConsents.push(...cdcConsents);
```

Reply:
done